### DISCUSSION TITLE
Welcome to ScreenToGif Discussions!

### DISCUSSION THREAD

**Discussion by NickeManarin** [NickeManarin/ScreenToGif#984] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by BloggerJen** [comment]
Hi! I get an on screen error to change the save file handling so that I can overwrite same file names. Can you tell me where to find that setting? I've poked around and I haven't been able to find it. Thanks!

**Reply by NickeManarin** [reply]
Hi, do you mean this?

![image](https://user-images.githubusercontent.com/14798947/145323183-928d241f-97d8-48ef-840c-42c2c60a7c1a.png)

**Reply by BloggerJen** [reply]
Yesssss!! Thank you!!
That section was collapsed on my screen and I didn't realize it. Oh my  gosh. I looked for it on three separate occasions, too.
